conans/client/loader.py

```diff
@@ -42,16 +42,20 @@ def __init__(self, runner, output, python_requires):
         self._python_requires = python_requires
         sys.modules["conans"].python_requires = python_requires
 
-    def load_class(self, conanfile_path):
-        with self._python_requires.capture_requires() as python_requires:
-            loaded, filename = parse_conanfile(conanfile_path)
+    @staticmethod
+    def load_class_static(python_requires, conanfile_path):
```

Comment:
I agree with factoring the repeated code. Just it might not be in the ConanFileLoader but a function. I see it at the level of ``parse_conanfile`` and ``parse_module``. Seems ``parse_conanfile`` could be made private and replaced for this one as public one?